pkg/api/v1alpha2/api.proto

```diff
@@ -439,9 +451,7 @@ message DeleteTrialReply {
 
 message GetTrialListRequest {
     string experiment_name = 1;
-    string start_time = 2; ///The start of the time range. RFC3339 format
-    string end_time = 3; ///The end of the time range. RFC3339 format
-    string filter = 4;
+    string filter = 2;
```

Comment:
what does `filter` look like? I don't find it is used in GetTrialList()

Reply:
Sorry, I forget it. I will add it to GetTrialList.

Reply:
Since the filter is used to filter trial name, can we rename it to filter_by_name? Also in db function, please handle `filter == ""` case.

Reply:
Thanks! updated.